<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="register_user_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>260</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton{
background-color: rgba(2, 65, 118, 255);
color: rgba(255, 255, 255, 200);
border-radius: 5px; 
}
QPushButton#pushButton:pressed{
padding-left: 5px;
padding-top: 5px;
background-color: rgba(2, 65, 118, 255);
background-position: calc(100% - 10px)center;
}
QPushButton#pushButton{
background-color: rgba(2, 65, 118, 200);
}</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_main_window">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Main Window</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_is_superuser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0 0);
font: 14pt &quot;MS UI Gothic&quot;;
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 255, 255);
padding-bottom:7px</string>
   </property>
   <property name="text">
    <string>is_superuser</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_field">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0 0);
font: 14pt &quot;MS UI Gothic&quot;;
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 255, 255);
padding-bottom:7px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password1_field">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>260</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0 0);
font: 14pt &quot;MS UI Gothic&quot;;
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 255, 255);
padding-bottom:7px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password2_field">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>260</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0 0);
font: 14pt &quot;MS UI Gothic&quot;;
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 255, 255);
padding-bottom:7px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(16, 30, 41, 240);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>18</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="error_field">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;News701 BT&quot;; color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>register_user_button</zorder>
  <zorder>back_to_main_window</zorder>
  <zorder>check_is_superuser</zorder>
  <zorder>username_field</zorder>
  <zorder>password1_field</zorder>
  <zorder>password2_field</zorder>
  <zorder>error_field</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
